extensions/kubernetes-client/deployment/src/main/java/io/quarkus/kubernetes/client/deployment/DevServicesKubernetesProcessor.java

```diff
@@ -92,7 +93,8 @@ public DevServicesResultBuildItem setupKubernetesDevService(
             CuratedApplicationShutdownBuildItem closeBuildItem,
             LoggingSetupBuildItem loggingSetupBuildItem,
             DevServicesConfig devServicesConfig,
-            BuildProducer<KubernetesDevServiceInfoBuildItem> devServicesKube) {
+            BuildProducer<KubernetesDevServiceInfoBuildItem> devServicesKube,
+            BuildProducer<KubernetesDevServiceRequestBuildItem> devServiceKubeRequest) {
```

Comment:
```suggestion
            Optional<KubernetesDevServiceRequestBuildItem> devServiceKubeRequest) {
```
KubernetesDevServiceRequestBuildItem is meant to be used by other extension to let this extension know that we want to start the dev service of the particular flavor. Think of it as an alternative to configuration for enabling this dev sevice that extensions can use.